nixpkgs_review/review.py

```diff
@@ -272,15 +274,19 @@ def build_pr(self, pr_number: int) -> dict[System, list[Attr]]:
             packages_per_system = self.github_client.get_borg_eval_gist(pr)
         else:
             packages_per_system = None
-        merge_rev, pr_rev = fetch_refs(
-            self.remote,
-            pr["base"]["ref"],
-            f"pull/{pr['number']}/head",
-        )
 
         if self.checkout == CheckoutOption.MERGE:
-            base_rev = merge_rev
+            base_rev, pr_rev = fetch_refs(
+                self.remote,
+                pr["base"]["ref"],
+                f"pull/{pr['number']}/merge",
```

Comment:
In fact there is a race condition with this ref sometimes as github needs some time to compute it. But this is probably only relevant for CIs. So we should be unlikely to when humans run nixpkgs-review (except for @SuperSandro2000 maybe)